_sass/1-modules/_mixins.scss

```diff
@@ -0,0 +1,84 @@
+// SASS mixins =================================================================
+
+@mixin box-shadow($shadow) {
+  box-shadow: $shadow;
+}
+
+@mixin border-radius($radius) {
+  border-radius: $radius;
+  background-clip: padding-box;
+}
+
+@mixin transition($transition) {
+  transition: $transition;
+}
+
+@mixin box-sizing($type) {
+  box-sizing: $type;
+}
+
+@mixin animate($animation, $speed) {
+  animation-duration: $speed;
+  animation-fill-mode: both;
+  animation-name: $animation;
+}
+
+@mixin siteWidth {
+  width: 720px;
+
+  @include breakpoint(s) {
+    width: calc(100vw - 64px);
+  }
+
+  @include breakpoint(xs) {
+    width: calc(100vw - 48px);
+  }
+}
+
+
+// Responsive Layout Scheme ====================================================
+
+// Spacing Variables
+$pad: 30px; // default and standard
+$pad_: 20px; // tablet and mobile
+
+
+// whole values
+$x_1: 8px;
+$x_2: 16px;
+$x_3: 24px;
+$x_4: 32px;
+$x_5: 40px;
+$x_6: 48px;
+$x_7: 56px;
+$x_8: 64px;
+
+// half values - use only when you really cant work with an 8 value
+$x_half: 2px;
+$x_1half: 4px;
+$x_2half: 12px;
+$x_3half: 20px;
+$x_4half: 28px;
+$x_5half: 36px;
+$x_6half: 44px;
+$x_7half: 52px;
+
+
+
```

Comment:
Can this be removed in favor of dividing one of the main spacer variables? 🤔

Reply:
actually, I havent needed this at all. I can just remove it